cogs/custom.py

```diff
@@ -2,17 +2,18 @@
 import time
 
 from discord import Color, Embed, Member
-from discord.ext.commands import Bot, Cog, Context, command, has_permissions
+from discord.ext.commands import BadArgument, Bot, Cog, Context, command, has_permissions
 
 
 class Custom(Cog):
     def __init__(self, bot: Bot) -> None:
         self.bot = bot
 
+    # Commands
```

Comment:
```suggestion
```
This shouldn't be here. That comment is useless as it's pretty obvious that these are commands.